great_expectations/checkpoint/checkpoint.py

```diff
@@ -154,7 +161,7 @@ def run(
             raise CheckpointRunWithoutValidationDefinitionError()
 
         if not self.id:
-            self._add_to_store()
+            self.save()
```

Comment:
Should we save each time no matter what? Is there a possibility of persisting a checkpoint and not having the children persisted? I think so right?

Reply:
Yes, we should persist each time.